server/api/actions/users.js

```diff
@@ -5,12 +5,18 @@ const getUsers = async () => {
   return data;
 };
 
-const createUser = async (name) => {
-  const { rows: data } = await db.execute('server/sql/users/put.sql', {name});
+const createUser = async ({name, identifier}) => {
+  const { rows: data } = await db.execute('server/sql/users/put.sql', {name, identifier});
   return data;
 };
 
+const findUser = async (identifier) => {
+  const { rows: data } = await db.execute('server/sql/users/findUser.sql', {identifier});
```

Comment:
I'm used to the file naming being something off of `GET`, `PUT`, `PATCH`, etc. So in this situation, I think it's fine to keep the naming as you have it, but if you want to be more specific and follow the "http methods" nomenclature then you could name it as `server/sql/users/getByIdentifier.sql`. This way in the future if we say wanted to find a user instead by their name or phone number or something else (not necessarily a realistic scenario) then we could do `server/sql/users/getByName.sql`